script.js

```diff
@@ -13,6 +13,189 @@ xMenu.addEventListener('click', () => {
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    short_desciption:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    tags: ['HTML', 'Ruby on Rails', 'CSS', 'JavaScript'],
+    live_link: '',
+    source_link: '',
+  },
+];
+
+const works = document.getElementById('works');
+for (let i = 0; i < projects.length; i += 1) {
+  let tagsList = '';
+  for (let j = 0; j < projects[i].tags.length; j += 1) {
+    tagsList += `<li>${projects[i].tags[j]}</li>`;
+  }
+  const card = document.createElement('div');
+  card.innerHTML = `${`<div class="card">
+          <div class="pv `
+    + 'pv'}${
+    i + 1
+  }">
+            <img
+              class="preview"
+              src="${
+  projects[i].mobile_img
+}"
+              alt="Website Preview"
+            />
+          </div>
+          <div class="card_details">
+            <h2>${
+  projects[i].name
+}</h2>
+            <div class="type_and_date">
+              <span>${
+  projects[i].info[0]
+}</span>
+              <span>●</span>
+              <span>${
+  projects[i].info[1]
+}</span>
+              <span>●</span>
+              <span>${
+  projects[i].info[2]
+}</span>
+            </div>
+            <p>${
+  projects[i].short_desciption
+}</p>
+            <ul class="prog_lang">${
+  tagsList
+}</ul>
+            <button type="button" id="see_project_${
+  i + 1
+}">See Project</button>
+          </div>
+        </div>`;
+  works.appendChild(card);
+}
+
+function createPopup(cardNo) {
+  let tagsList = '';
+  for (let j = 0; j < projects[cardNo].tags.length; j += 1) {
+    tagsList += `<li>${projects[cardNo].tags[j]}</li>`;
+  }
+  const popupWindow = document.createElement('div');
+  popupWindow.innerHTML = `<div id="popup_container" class="popup_container">
+<div class="popup_card">
+  <div class="popup_title">
+    <img id="x-popup" src="./images/x-dark.png" alt="">
+    <h2>${
+  projects[cardNo].name
+}</h2>
+    <div class="info">
+      <span>${
+  projects[cardNo].info[0]
+}</span>
+      <span>●</span>
+      <span>${
+  projects[cardNo].info[1]
+}</span>
+      <span>●</span>
+      <span>${
+  projects[cardNo].info[2]
+}</span>
+    </div>
+  </div>
+  <div class="popup_pv `
+    + `pv${
+      cardNo + 1
+    }">
+    <img
+      class="preview"
+      src="${
+  projects[cardNo].mobile_img
+}"
+      alt="Website Preview"
+    />
+  </div>
+  <div class="popup_details">
+  <p>${
+  projects[cardNo].description
+}</p>
+    <div>
+      <ul class="prog_lang">${
+  tagsList
+}</ul>
+      <hr>
+      <div class="popup_btn">
+        <button type="button" href="${
+  projects[cardNo].live_link
+}">See Live <i class="material-icons">arrow_forward</i></button>
+        <button type="button" href="${
+  projects[cardNo].source_link
+}">See source<i class="fa fa-github"></i></button>
+      </div>
```

Comment:
Please follow the design guideline on Figma and ensure that icons on the See Live and See Source are identical 👍  You can export as SVG from Figma.
<img width="471" alt="Screenshot 2023-04-11 at 10 45 26 PM" src="https://user-images.githubusercontent.com/25887253/231274649-4fd44718-5127-4797-8615-68dc5b17e15c.png">
<img width="386" alt="Screenshot 2023-04-11 at 10 45 37 PM" src="https://user-images.githubusercontent.com/25887253/231274654-a3db96a9-6b69-4763-8dc6-0f667f2ec4ef.png">